src/section-exploring-two-variable-data-and-rate-of-change.ptx

```diff
@@ -807,7 +807,7 @@
             </image>
         </figure>
 
-        <p>Whenever the rate of change is constant no matter which two <m>(x,y)</m>-pairs (or data pairs) are chosen from a data set, then you can conclude the graph will be a straight line <em>even without making the graph</em>. For obvious reasons, we call this kind of relationship a <term>linear</term> relationship. We'll study linear relationships in more detail throughout this chapter. Right now in this section, we feel it is important to simply identify if data has a linear relationship or not.</p>
+        <p>Whenever the rate of change is constant no matter which two <m>(x,y)</m>-pairs (or data pairs) are chosen from a data set, then you can conclude the graph will be a straight line <em>even without making the graph</em>. For obvious reasons, we call this kind of relationship a <term>linear</term> relationship. We'll study linear relationships in more detail throughout this chapter. Right now in this section, we feel it is important to simply identify if data have a linear relationship or not.</p>
```

Comment:
Ross, please read: http://www.onlinegrammar.com.au/top-10-grammar-myths-data-is-plural-so-must-take-a-plural-verb/

And then please stop changing these verb conjugations, because I just change them back :)